### ISSUE TITLE
checklists have to be edited manually to mark complete items

### ISSUE THREAD

**Issue by JamesGKent** [gogits/gogs#4967]
currently running gogs 0.11.34 on raspbian jessie.  
on github if a checklist is created in an issue items in it can be marked as complete by clicking the rendered checkbox, however I seem to have to edit the post/comment and add an x in the box manually instead.

**Comment by Fastidious** [comment]
@JamesGKent yes, that is the Markdown way. I am not sure how GitHub does it, but Gogs does it differently. Perhaps phrasing this as a request for enhancement, and not as an issue, would be best?

**Comment by JamesGKent** [comment]
sorry, i had that in the title and it was a bit long didn't realize that i hadn't added it back to the body. it should be added as a label but i'm unable to do so.
